status-im/status-react
# Fix channel-action component

fixes #19021

### Summary

For the `channel-action` component was fixed according to the [design system](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=866-26242&mode=design&t=8DbWIrF1gaxVFo7V-4):
1. The color when button is pressed in
2. The big button is using flexbox now instead of fixed width, to assure it looks right on all screens
3. Separated `channel-action` from `channel-actions`, according to the design system
4. Added preview screen for `channel-action`

#### Platforms


- Android
- iOS


#### Areas that maybe impacted


##### Functional

- 1-1 chats
- public chats
- group chats


### Steps to test


- Open Status
- Open a chat
- Press the channel action (button should have a different color)
- (or go to preview -> community -> channel-action(s))

status: ready
